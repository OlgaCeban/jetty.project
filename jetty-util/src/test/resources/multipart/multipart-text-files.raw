--ny9C5eIZL7fsfPY9ONPy8Lxb6tkgEv1
Content-Disposition: form-data; name="text"
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

text default

--ny9C5eIZL7fsfPY9ONPy8Lxb6tkgEv1
Content-Type: text/plain; charset=UTF-8
X-SHA1: 588A0F273CB5AFE9C8D76DD081812E672F2061E2
Content-Disposition: form-data; name="file1"; filename="a.txt"
Content-Transfer-Encoding: binary

Content of a.txt

--ny9C5eIZL7fsfPY9ONPy8Lxb6tkgEv1
Content-Type: text/html; charset=UTF-8
X-SHA1: 9A9005159AB90A6D2D9BACB5414EFE932F0CED85
Content-Disposition: form-data; name="file2"; filename="a.html"
Content-Transfer-Encoding: binary

<!DOCTYPE html><title>Content of a.html.</title>
--ny9C5eIZL7fsfPY9ONPy8Lxb6tkgEv1--
