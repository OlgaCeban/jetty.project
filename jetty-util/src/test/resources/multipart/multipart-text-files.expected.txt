Content-Type|multipart/form-data; boundary="ny9C5eIZL7fsfPY9ONPy8Lxb6tkgEv1"
Parts-Count|3
Part-ContainsContents|text|text default
Part-ContainsContents|file1|Content of a.txt
Part-ContainsContents|file2|<!DOCTYPE html><title>Content of a.html
Part-Filename|file1|a.txt
Part-Filename|file2|a.html
Part-Sha1sum|file1|588A0F273CB5AFE9C8D76DD081812E672F2061E2
Part-Sha1sum|file2|9A9005159AB90A6D2D9BACB5414EFE932F0CED85